runtime/onert/core/src/loader/CircleLoader.cc

```diff
@@ -71,6 +74,8 @@ class CircleLoader final : public loader::BaseLoader<LoaderDomain>
   void loadRmsNorm(const Operator *op, ir::Graph &subg);
   void loadRoPE(const Operator *op, ir::Graph &subg);
   void loadCall(const Operator *op, ir::Graph &subg);
+  void loadRunModel(const Operator *op, ir::Graph &subg);
+  void loadCustomCircle(const Operator *op, ir::Graph &subg);
```

Comment:
@hseok-oh I expected `loadCustom`: `load` + opname. Why do we need to add `Circle`?

Reply:
I tried, but `BaseLoader` already have `loadCustom` as private and `CircleLoader` cannot have same name with `BaseLoader`.

Reply:
I updated